armi/physics/neutronics/parameters.py

```diff
@@ -723,23 +724,23 @@ def _getNeutronicsCoreParams():
     with pDefs.createBuilder(categories=[parameters.Category.neutronics]) as pb:
         pb.defParam(
             "eigenvalues",
-            units=None,
+            units=units.UNITLESS,
             description="All available lambda-eigenvalues of reactor.",
```

Comment:
That will require a ticket though, because the name of this parameter is so generic that it makes it hard to grep through downstream repos to find it.

It might be easier to just remove the Parameter, and see if anything breaks downstream.